cme/modules/enum_av.py

```diff
@@ -3,106 +3,102 @@
 
 # All credit to @an0n_r0
 # project : https://github.com/tothi/serviceDetector
+# Module by @mpgn_x64
 
-from impacket.dcerpc.v5 import lsat, lsad
+from impacket.dcerpc.v5 import lsat, lsad, transport
 from impacket.dcerpc.v5.dtypes import NULL, MAXIMUM_ALLOWED, RPC_UNICODE_STRING
-from impacket.dcerpc.v5 import transport
 import pathlib
 
-
 class CMEModule:
-    """
-    Uses LsarLookupNames and NamedPipes to gather information on all endpoint protection solutions installed on the the remote host(s)
-    Module by @mpgn_x64
-    """
-
+    
     name = "enum_av"
-    description = "Gathers information on all endpoint protection solutions installed on the the remote host(s) via LsarLookupNames (no privilege needed)"
+    description = "Gathers information on all AV/EDR solutions installed on the the remote host"
     supported_protocols = ["smb"]
     opsec_safe = True
     multiple_hosts = True
 
     def __init__(self, context=None, module_options=None):
         self.context = context
         self.module_options = module_options
-
+        
     def options(self, context, module_options):
-        """ """
+        """Placeholder for module options."""
```

Comment:
Fill this in?

Reply:
there are no options at this point but I might introduces a few in the future.